build.zig

```diff
@@ -4,6 +4,8 @@ const fs = std.fs;
 const mem = std.mem;
 const zbs = std.build;
 
+const version = "0.1.0-dev";
```

Comment:
This could use a doc comment explaining when/how to change this when doing a release.